src/i18n/messages.json

```diff
@@ -74,6 +74,90 @@
         "description": "Globetrotter (covid permettendo), amante delle bibite gassate e climber. Amo le sfide. Mi piace studiare quello che mi interessa. Sono sintetico.",
         "website": "",
         "permalink": "nicpuppa"
+      },
+      "Agny": {
+        "name": "Angela Busato",
+        "github": "AngyDev",
+        "github_url": "https://github.com/AngyDev",
+        "linkedin_url": "https://www.linkedin.com/in/angela-busato-91145ab1/",
+        "twitter_url": "https://twitter.com/angela_busato",
+        "image": "/img/_dev/Agny.jpg",
+        "secondary_image": "/img/_dev/Agny.jpg",
+        "description": "Sono un'esploratrice, mi piace la ricerca e la sperimentazione negli ambiti più disparati, programmazione, fotografia, stampa 3D, e fai da te sono i primi della mia lista. Nel tempo libero mi piace viaggiare e conoscere nuovi posti, persone, culture ma soprattutto conoscere il cibo, ci sono sempre cose nuove da scoprire.",
+        "website": "",
+        "permalink": "AgnyDev"
+      },
+      "DanerSound": {
+        "name": "Andre Cristhian",
+        "github": "DannerSound",
+        "github_url": "https://github.com/DannerSound",
+        "linkedin_url": "https://www.linkedin.com/in/angela-busato-91145ab1/",
```

Comment:
This is not the right LinkedIn url of DanerSound, please use the correct one

Reply:
done